src/main/scala/ai/eto/rikai/sql/model/fs/FileSystemRegistry.scala

```diff
@@ -41,8 +41,12 @@ class FileSystemRegistry(val conf: Map[String, String])
     * @return [[Model]] if found.
     */
   @throws[ModelNotFoundException]
-  override def resolve(uri: String, name: Option[String]): Model = {
+  override def resolve(
+      uri: String,
+      name: Option[String],
+      options: Option[Map[String, String]]
+  ): Model = {
     logger.info(s"Resolving ML model from ${uri}")
-    Python.resolve(pyClass, uri, name, Map.empty)
+    Python.resolve(pyClass, uri, name, options.getOrElse(Map.empty))
```

Comment:
Do you think we should actually pass a `ModelSpec` from jvm to python, merged in python, and return a Model back?

Reply:
Is this because you're thinking about reading the ModelSpec out the catalog db and then passing it to python to be "merged" and turned into an actual Model instance? or this is because of some other reason?

Reply:
I see that we will have more parameters than these three in the future. Using an interface ModelSpec here can make the code more stable

Reply:
we dont need to change this now.